src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -3083,23 +3088,35 @@ def persistent_topic_stats(conn, endpoint_type):
     topic_conf.del_config()
     # delete the bucket
     conn.delete_bucket(bucket_name)
-    receiver.close(task)
+    if endpoint_type == 'http':
+        receiver.close()
+    elif endpoint_type == 'amqp':
+        stop_amqp_receiver(receiver, task)
+    elif endpoint_type == 'kafka':
+        receiver.close(task)
```

Comment:
```
receiver.close(task)
```
The above code should work because close function is defined in all of the endpoints:
http endpoint: https://github.com/ceph/ceph/blob/a078c3c9d9c38653901f8f4bb982bfba1a5912c5/src/test/rgw/bucket_notification/test_bn.py#L263-L277

kafka endpoint:
https://github.com/ceph/ceph/blob/a078c3c9d9c38653901f8f4bb982bfba1a5912c5/src/test/rgw/bucket_notification/test_bn.py#L452-L453

ampq endpoint:
https://github.com/ceph/ceph/blob/a078c3c9d9c38653901f8f4bb982bfba1a5912c5/src/test/rgw/bucket_notification/test_bn.py#L347-L348